packages/synapse-react-client/src/components/PortalFeatureHighlights/PortalFeatureHighlights.tsx

```diff
@@ -0,0 +1,83 @@
+import { Typography, Box, Button, CardMedia, Grid } from '@mui/material'
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+export type PortalFeatureHighlightsProps = {
+  reverseOrder?: boolean
+  title?: string
+  image?: string
+  buttonText?: string
+  summaryText?: string | React.ReactNode
```

Comment:
I added a ReactNode option just in case someone wants to pass in stylized text like in the images where "Cavatica" is stylized.